### DISCUSSION TITLE
pgbouncer flag

### DISCUSSION THREAD

**Discussion by byeong0430** [prisma/prisma#20877] [Q&A] [answered]
### Question

Hey guys, I'm new to Prisma and have a question regarding pgbouncer.

This article makes it sound like all I need to do is to add `pgbouncer=true` flag:

https://www.prisma.io/docs/guides/performance-and-optimization/connection-management/configure-pg-bouncer

Is it true? Or do I need to set up my own pgbouncer and then connect to it with the flag?

Thanks in advance :) 

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by janpio** [comment]
Your connection string indeed needs to point to a PgBouncer instance. The article you linked to only describes how to _configure_ Prisma to use a connection string for PgBouncer.

**Reply by ludralph** [reply, marked as answer]
Hi @byeong0430 👋 

In addition, you need to set up your own PgBouncer and connect it with the flag on the connection string. You can take a look at this [video](https://youtu.be/tsOHw2T1HvI?t=456) that shows how to set up the PgBouncer instance on Digital Ocean, and as you watch along, you will see how the flag is connected [here](https://youtu.be/tsOHw2T1HvI?t=827).

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by byeong0430** [comment]
Thank you so much guys :)

**Reply by ludralph** [reply]
You're welcome 🙌
